bin/dummy_cuckoo_api.py

```diff
@@ -108,18 +126,4 @@ def post(self):
 
 
 if __name__ == '__main__':
-    PARSER = reqparse.RequestParser()
```

Comment:
Was reqparse unnecessary or where did it go?

Reply:
It was a left over

Reply:
Good riddance. Cheers. :)